doc/changelog/02-specification-language/11235-non_maximal_implicit.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  Syntax for non maximal implicit arguments in toplevel definitions using
```

Comment:
Does this mean that the syntax is not valid underneath forall/fun?  If so, why this restriction?  If not, the changelog should not imply that it's only valid for top-level definitions.

Reply:
You are right, it works everywhere!

Reply:
I'm not sure it works properly everywhere. Let's confirm it is the case before extending the new syntax to local binders.